spec/gem_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+RSpec.describe Rootstrap::Gem do
+  gem_name = 'test'
+
+  describe '#create' do
+    before(:all) { described_class.new(gem_name: gem_name).create }
+    after(:all) { `rm -rf ./#{gem_name}` }
+
+    let(:list_files) { `ls -a`.split("\n") }
+    let(:list_files_gem) { `ls -a #{gem_name}`.split("\n") }
+    let(:gemspec) { File.open("./#{gem_name}/#{gem_name}.gemspec").read }
+
+    it 'creates a new folder' do
+      expect(list_files).to include gem_name
+    end
+
+    it 'adds the default gems to the gemspec' do
+      expect(gemspec).to include 'rake'
```

Comment:
maybe you can

```ruby
expect(gemspec).to eq(<~ HEREDOC
  ...
HEREDOC)
```

to make sure the Gemfile is correctly formatted too.

For example a Gemfile with content `rake rspec rubocop 'simplecov', '~> 0.17.1'` would pass all the tests but would be wrong